### ISSUE TITLE
🐛 Link colour of Delete process is not red

### ISSUE THREAD

**Issue by mnigh** [GCTC-NTGC/gc-digital-talent#8461]
## 🐛 Bug

Link colour of **Delete process** is not red even though action is negative and destructive.

## 📸 Screenshot

![Screen Shot 2023-11-09 at 12 09 10](https://github.com/GCTC-NTGC/gc-digital-talent/assets/3046459/255f2509-970d-4527-92b4-d207414a63d7)

## 🙋‍♀️ Proposed Solution

Change link to red.

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the bug was addressed.

- [ ] link to **Delete process** is red

**Comment by mnigh** [comment]
@GCTC-NTGC/designers is this something you are able to address?

**Comment by Jerryescandon** [comment]
🤔 Unsure, I'm combing through the designs to refresh my memory, and I think I was assuming we were handling 'destructive-actions' different from 'destructive-paths'. e.g:
- destructive-action: an action that causes irreversible changes to the data. (Error color)
- destructive-path: an action that leads to a destructive action. (Warning color????)

But now that you bring it up I'm not sure if we have a standard approach for destructive path or if we are even considering it at all.

An additional issue is that our tertiary and quaternary colours are too close to the stoplight colors 😖 

I will bring it to the desfam channel on slack for more input

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/16977254/e07e1ffb-4f79-43f2-8205-2779d799aab3)

**Comment by gobyrne** [comment]
Ok, designers are ok with it. Let's leave

**Comment by substrae** [comment]
Oof, okay, there are a handful of inconsistencies here:
* We should be using `error` for anything destructive, even if there is a confirmation
* We should never be using `tertiary`, `quaternary`, or `quinary` for anything that isn't directly tied to design instances assigned to those colours (e.g. specific cards on the site); always default to using `error`, `warning`, and `success` for actionable tasks
* The button alignment in the confirmation dialog - our button layouts need to be standardized ASAP - primary actions should always be first - in this case, even though the primary action is destructive, it's a confirmation of the destructive action, so as the main action being taken, it should be first (with Cancel following)

**Comment by RM-1978** [comment]
@substrae,  Given that this issue affects cognitive accessibility , should we prioritize this issue to in the Design repository for decision?

**Comment by substrae** [comment]
Fixing this one issue is straightforward enough, expanding the scope to the rest of the site is opening a can of worms I'm not sure we have time for at the moment. If we want to get this one fixed, sure. Direction on color and layout for button groups can be added to the Style guide in Figma. Otherwise we need a new ticket for auditing the rest of the site, and it probably won't get addressed in the short-term.

**Comment by RM-1978** [comment]
@gobyrne can we change the colour of this instance of the  Delete process  and update the style guide so that it is red given that the action is negative and destructive.
